migration-tools/migration/http_docublocks.py

```diff
@@ -36,12 +36,13 @@ def str_presenter(dumper, data):
 
 
 def initBlocksFileLocations():
-    with open(APIDOCS_FILE, 'r') as apiDocs:
+    with open(APIDOCS_FILE, 'r', encoding="utf-8") as apiDocs:
         data = apiDocs.read()
 
         docuBlocks = re.findall(r"<!-- filename: .* -->\n@startDocuBlock .*", data)
         for docuBlock in docuBlocks:
             fileLocation = re.findall(r"(?<=<!-- filename: ).*(?= -->)", docuBlock)[0]
+            fileLocation = fileLocation.replace("/work/ArangoDB/", "C:/ArangoDB/arangodb/") # HACK
```

Comment:
I think `migration.py` should have a secondary `--src` parameter for setting/overriding the path to the local working copy of the main repo.